app/presenters/admin_auction_status_presenter/available.rb

```diff
@@ -4,7 +4,26 @@ def header
   end
 
   def body
-    I18n.t('statuses.bid_status_presenter.available.admin.body', end_date: end_date)
+    if total_bids > 0
+      I18n.t('statuses.bid_status_presenter.available.admin.has_bids',
+             end_date: end_date,
+             winner_url: winner_url,
+             total_bids: total_bids
+            )
+    else
+      I18n.t('statuses.bid_status_presenter.available.admin.body', end_date: end_date)
+    end
+  end
+
+  private
+
+  def total_bids
+    auction.bids.count
+  end
+
+  # winning_bid is a NullBid when sealed-bid auctions are available
```

Comment:
yeah I think what you are doing here makes total sense, I think naming the method something like `unmasked_winner_name_for_admin` would be an improvement over having a comment 💥